src/interactor/state_machine/interactor_steps/LoadStep.cc

```diff
@@ -26,20 +26,13 @@ void LoadStep::ChangeStep(std::shared_ptr<Step> &active_step) {
 
 template <>
 std::unique_ptr<Command> LoadStep::HandleStage<1>(Context &) {
-  if (arg_.empty()) {
-    return ReportError(Strings::kMultipleArgumentDoesNotSupported);
-  }
-  filename_ = validator_->ConsumeOneTokenFrom(arg_);
-  if (!arg_.empty()) {
-    return ReportError(Strings::kMultipleArgumentDoesNotSupported);
-  }
-  return std::make_unique<LoadTasksFromFileCommand>(filename_);
+  return std::make_unique<LoadTasksCommand>();
 }
 
 template <>
 std::unique_ptr<Command> LoadStep::HandleStage<0>(Context &ctx) {
```

Comment:
I don't understand why it is still a template.
Will approve the PR, to see the whole picture